ctapipe/pipeline/algorithms/hessio_reader.py

```diff
@@ -0,0 +1,35 @@
+from ctapipe.utils.datasets import get_path
+from ctapipe.io.hessio import hessio_event_source
+from ctapipe.configuration.core import Configuration, ConfigurationException
+import threading
+from ctapipe.core import Component
+from traitlets import Unicode
+
+
+class HessioReader(Component):
```

Comment:
I'd rename this `SimTelArrayReader` (we should start to rename things to avoid the HESS name)